src/Products/GenericSetup/ZCatalog/exportimport.py

```diff
@@ -13,11 +13,10 @@
 """ZCatalog export / import support.
 """
 
+from Products.ZCatalog.interfaces import IZCatalog
 from zope.component import adapts
 from zope.component import queryMultiAdapter
 
-from Products.ZCatalog.interfaces import IZCatalog
-
```

Comment:
This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.